_styles.scss

```diff
@@ -1,2 +1,2 @@
-@import "variables/variables";
-@import "node_modules/@unumux/willow/styles";
\ No newline at end of file
+@import "node_modules/@unumux/willow/utilities/functions/functions";
```

Comment:
added b/c 
- `_spacing.scss` uses the `space()` function
- `_fonts.scss` uses the `line-height()` function
- `_text-styles` uses the `line-height()` function